packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -869,13 +875,13 @@ export const prepareVaultManagerKit = (
               transfers.length,
             );
 
-            facets.helper.markRestoreDebt(
-              AmountMath.subtract(totalDebt, debtRemaining),
-            );
             facets.helper.sendToReserve(collatRemaining, liqSeat);
             facets.helper.markDoneLiquidating(totalDebt, totalCollateral);
+            facets.helper.recordShortfallAndProceeds({
+              overage: accounting.overage,
```

Comment:
without knowing the types, reviewing in web UI it looks like this could be missing some properties.
```suggestion
              ...accounting,
```